python/ray/_private/state_api_test_utils.py

```diff
@@ -319,11 +319,7 @@ def periodic_invoke_state_apis_with_actor(*args, **kwargs) -> ActorHandle:
 
 def get_state_api_manager(gcs_address: str) -> StateAPIManager:
     gcs_aio_client = GcsAioClient(address=gcs_address)
-    gcs_channel = GcsChannel(gcs_address=gcs_address, aio=True)
-    gcs_channel.connect()
-    state_api_data_source_client = StateDataSourceClient(
-        gcs_channel.channel(), gcs_aio_client
-    )
+    state_api_data_source_client = StateDataSourceClient(gcs_aio_client)
```

Comment:
does all client share the same channel automatically? I think this is important for scalabilty.

Reply:
yes all client share a same grpc channel